custom/enikshay/case_utils.py

```diff
@@ -35,12 +35,11 @@ def get_parent_of_case(domain, case_id, parent_case_type):
 
```

Comment:
yeah, it's a quirk of advanced modules I think. I think Simon fixed this here: https://github.com/dimagi/commcare-hq/pull/14501, but I'm not sure we have updated it everywhere yet.